### PR TITLE
hw_5_1_done

### PR DESCRIPTION
**Pull request by IrinaDyakonova**

### CODE REVIEW

File: src/main/java/core/basesyntax/Engine.java [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+                   int engineCapacity, int valvesPerCylinder, int power) {
+        this.enginesType = enginesType;
+        this.fuelGrade = fuelGrade;
+        this.engineCapacity = engineCapacity;
+        this.valvesPerCylinder = valvesPerCylinder;
+        this.power = power;
+    }
+
+    @Override
+    public Engine clone() throws CloneNotSupportedException {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Engine.java:21]: can we remove throws?

File: src/main/java/core/basesyntax/Car.java [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+        this.model = model;
+        this.color = color;
+        this.engine = engine;
+        this.driveUnit = driveUnit;
+        this.numberOfSeats = numberOfSeats;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+        return new Car(model, color, engine, driveUnit, numberOfSeats);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Car.java:20]: you will get a new Car but with the same engine

File: src/main/java/core/basesyntax/Engine.java (hunk 1 above)
Comment (IrinaDyakonova) [reply src/main/java/core/basesyntax/Engine.java:21]: yes, without it, too, passes, I did immediately without it, but then I looked at who they all accepted, so I added (

File: src/main/java/core/basesyntax/Car.java (hunk 2 above)
Comment (IrinaDyakonova) [reply src/main/java/core/basesyntax/Car.java:19]: ok

File: src/main/java/core/basesyntax/Engine.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/Engine.java]: we don't accept this task with throws)And it's bad to write off especially with no understanding

File: src/main/java/core/basesyntax/Car.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/Car.java:19]: that's not okey, that's problem. Imagine you have two cars but with one engine

**Pull request closed without merging**